search.html

```diff
@@ -4,86 +4,100 @@
-          <label class="search-label" for="search-title">include title</label>
-          <br />
-
-          <input type="checkbox" name="scope" id="search-content" value="content" checked/>
-          <label class="search-label" for="search-content">include content</label>
-          <br />
-
-          <input type="checkbox" name="scope" id="search-preview" value="preview" checked/>
-          <label class="search-label" for="search-preview">include preview</label>
-        </div>
-        <div class="col-md-3">
-          <div class="search-option-title">
-            keyword expansion
-            <i class="fa fa-question-circle search-tooltips"
-              rel="tooltip"
-              title="show search results even if keywords are partially available in the content"></i>
-          </div>
-          <input type="radio" name="expansion" value="expansion" checked> expand
-          <br>
-          <input type="radio" name="expansion" value="no-expansion"> do not expand
-        </div>
-        <div class="col-md-3">
-          <div class="search-option-title">search category
-            <i class="fa fa-question-circle search-tooltips"
-              rel="tooltip"
-              title="search within only certain types (default to all types)"></i>
-          </div>
-          <select name="type-facet"></select>
-        </div>
-        <div class="col-md-3">
-          <div class="search-option-title">filter by tags
-            <i class="fa fa-question-circle search-tooltips"
-              rel="tooltip"
-              title="search by tags"></i>
-          </div>
-          <select name="tag-facet" multiple="multiple" id="tag-facet"></select>
-        </div>
-
-        <!-- <div class="col-md-3">
-          <div class="search-option-title">keyword boolean</div>
-          <input type="radio" name="boolean" value="or" checked> OR (default)
-          <br>
-          <input type="radio" name="boolean" value="and"> AND
-        </div> -->
+    <div class="row text-left search-filter" id="search-tip" style="display: none;">
+      <div class="col-md-12">
+        <div class="search-option-tip">You can limit your search to page content or annotations and refine your annotation search by selecting tags.</div>
       </div>
-    <div class="media text-left" id="search-results"></div>
+    </div>
+
+    <div class="row text-left search-filter" id="type-filter" style="display: none;">
+      <div class="col-md-12">
+        <div class="search-option-title">filter by a type:</div>
+        <div id="facet-type"></div>
+      </div>
+    </div>
+
+    <div class="row text-left search-filter" id="tag-filter" style="display: none;">
+      <div class="col-md-12">
+        <div class="search-option-title">filter by a tag:</div>
+        <div id="facet-tag"></div>
+      </div>
+    </div>
+
+    <div class="row media text-left">
+      <div class="col-md-12" id="search-results"></div>
+    </div>
   </div>
 </div>
 
-
 <script type="text/javascript">
```

Comment:
Let's move the javascript logic into a separate js file, I think that will make it a bit easier to manage.